cloud/services/storage/keypair.go

```diff
@@ -0,0 +1,98 @@
+package storage
```

Comment:
Put in security instead ?
(Like cockpit ?)

Reply:
done (file moved in security)

Reply:
Ok